### ISSUE TITLE
[question] Beginner to python need to build then train a model with AdPredictor

### ISSUE THREAD

**Issue by RanaivosonHerimanitra** [ajtulloch/adpredictor#3]
Hi,
Just have a matrix of features X and a vector of labels y  then how can I tell your package to train the model?
Something like this:

AdPredictor(features=X, label=y) ?

But my first question is how to import /build the package?

Thks in advance,

**Comment by ajtulloch** [comment]
Hi @RanaivosonHerimanitra,

I wouldn't recommend using this package for anything serious (I would highly recommend `scikit-learn`, http://scikit-learn.org/).  That package has much better and more thorough implementations of many ML algorithms.

See https://github.com/ajtulloch/adpredictor/blob/master/example.py#L148-L154 for an example of how you this, in general

```
model = AdPredictor(config=AdPredictor.Config(...))
for (example_features, example_label) in zip(features, labels):
   model.train(example_features, example_label)
```

**Comment by RanaivosonHerimanitra** [comment]
Thks
